packages/react-native/src/primitives/Heading/Heading.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Text } from 'react-native';
+import { styles } from './styles';
+import { HeadingProps } from './types';
+
+export default function Heading({
+  children,
+  headingStyle,
+  level = 6,
+  truncated = false,
+}: HeadingProps): JSX.Element {
+  return (
+    <Text
+      accessibilityRole="header"
```

Comment:
Hmm maybe. I chose `"header"` because of [this](https://reactnative.dev/docs/accessibility#:~:text=header%20Used%20when%20an%20element%20acts%20as%20a%20header%20for%20a%20content%20section%20(e.g.%20the%20title%20of%20a%20navigation%20bar).).